packages/palette/src/elements/Input/Input.test.tsx

```diff
@@ -49,98 +49,52 @@ describe("Input", () => {
 
 describe("computeBorderColor", () => {
   it("defaults to returning black 10", () => {
-    const props = {
-      disabled: null,
-      error: null,
-    }
-    const color = computeBorderColor(props)
+    const color = computeBorderColor(null, null)
```

Comment:
Using unnamed arguments is a bit opaque -- can we preserve passing an object along?